.tests/cypress/_integration/add-to-cart.js

```diff
@@ -1,3 +1,4 @@
+// TODO: IDK what is this testing, would say that it should be removed
```

Comment:
This test checks adding a product to cart from catalog page using alpaca solr URL where the magento settings are setup to redirect user to product page after clicking on add to cart button in catalog page. So we have 2 scenarios for adding products to cart from catalog view (the second scenario was covered in `_integration/catalog/cart/add-to-cart-configurable.js`).